META.coq.in

```diff
@@ -517,6 +517,20 @@ package "plugins" (
     plugin(native)  = "string_notation_plugin.cmxs"
   )
 
+  package "numeral_notation" (
+    description = "Coq numeral notation plugin"
+    version 	= "8.13"
+
+    requires    = "coq.vernac"
```

Comment:
Can you remind me why this dependency is needed? (eg string_notation has no)

Reply:
I guess the question is for @ejgallego actually

Reply:
Seems like a bug in string_notation I think; the dependency is needed as these plugins use the vernac extension mechanism.

Reply:
Should I add that dependency to string_notation too?

Reply:
@alizter AFAICS that's the right thing to do [if in doubt check the `_build/install/default/lib/coq/META` file]